content/terraform-docs-common/docs/cloud-docs/api-docs/queries/index.mdx

```diff
@@ -0,0 +1,425 @@
+---
+page_title: /queries API reference for HCP Terraform
+description: Call the HCP Terraform API's `/queries` endpoint to create and manage query runs.
+tfc_only: true
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+# Query API reference
+
+Call query API endpoints to execute query runs against your HCP Terraform workspaces. Query runs retrieve information about workspace resources and configurations. They are asynchronous operations that you can monitor for status and results.
+
+## Overview
+
+The scope of the API includes the following endpoints:
+
+| Method   | Path                                              | Action                                                                                                     |
+|----------|-------------------------------------------------- |------------------------------------------------------------------------------------------------------------|
+| `POST`   | `/queries`                                        | Call this endpoint to [create a query run](#create-a-query-run).                                          |
+| `GET`    | `/queries/:query_id`                              | Call this endpoint to [show a query run](#show-a-query-run).                                              |
+| `POST`   | `/queries/:query_id/actions/cancel`               | Call this endpoint to [cancel a query run](#cancel-a-query-run).                                          |
+| `GET`    | `/workspaces/:workspace_id/queries`               | Call this endpoint to [list query runs for a workspace](#list-query-runs-for-a-workspace).                |
```

Comment:
nit: this ordering in this table doesn't reflect the actual ordering of the requests on the page. (list query runs is last here, but is 2nd on the page)

<img width="1616" height="726" alt="CleanShot 2025-10-31 at 11 27 09@2x" src="https://github.com/user-attachments/assets/48ea9f4b-afae-4e66-ad8b-4f5df623d718" />